db/migrate/20231201113835_fill_establishments_students_provider.rb

```diff
@@ -3,6 +3,8 @@
 class FillEstablishmentsStudentsProvider < ActiveRecord::Migration[7.1]
   def up
     Establishment.includes(:users).find_each do |establishment|
+      next if establishment.users.none? || establishment.users.first.provider.nil?
```

Comment:
pourquoi il n'y aurait pas de users du tout ?

Reply:
C'est l'erreur que tu m'as transféré hier. La migration ne passe pas en staging : https://sentry.incubateur.net/organizations/betagouv/issues/75649/?query=is%3Aunresolved+migrations&referrer=issue-stream&statsPeriod=14d